app/gui2/src/util/computedValueRegistry.ts

```diff
@@ -23,15 +26,22 @@ class ComputedValueDb extends ReactiveDb<ExpressionId, ExpressionInfo> {
 /** This class holds the computed values that have been received from the language server. */
 export class ComputedValueRegistry {
   public db = new ComputedValueDb()
+  private dataflowErrorVisualizationIds = new Map<ExpressionId, Uuid>()
   private _updateHandler = this.processUpdates.bind(this)
   private executionContext: ExecutionContext | undefined
+  private visualizationDataRegistry: VisualizationDataRegistry | undefined
```

Comment:
I think this is mixing responsibilities. `ComputedValueRegistry` is about expression updates received from the engine. It should not handle data flow error visualizations. Let's make a separate util/store for it.